notebooks/L1_working_with_age_ensembles.ipynb

```diff
@@ -25,9 +25,10 @@
     "\n",
```

Comment:
<p>this may be a ReviewNB artifact, but it looked like the same exact cell had just been executed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LinkedEarth/PyleoTutorials/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
I think this is a reviewnb artifact or something because I can't see any redundancies in the notebook.